specification/common-types/resource-management/v6/privatelinks.json

```diff
@@ -2,34 +2,35 @@
   "swagger": "2.0",
   "info": {
     "title": "Common types",
-    "version": "6.0"
+    "version": "v6"
   },
   "paths": {},
   "definitions": {
     "PrivateEndpoint": {
       "type": "object",
-      "description": "The private endpoint resource.",
+      "description": "The Private Endpoint resource.",
```

Comment:
Isn't capitalizing it as 'private endpoint' fine? It's not a brand name. I think its analogous to 'The virtual machine resource'.